### PR TITLE
add meta service page

### PR DESCRIPTION
**Pull request by zhoujinsong**
implement #101

### CODE REVIEW

**Review by majin1102 — CHANGES_REQUESTED** [review]
leave a blank between Chinese and English single word

File: site/docs/ch/meta-service/dashboard.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+```
+
+???+ 注意
+
+    修改配置文件后需重启AMS服务才可生效，参考[启动/重启/关闭](#_3)。
+
+### 使用Flink执行结构优化
+
+在默认的AMS配置中，我们已经初始化了一个名为`default`的optimize group，它会在AMS本地新启动一个进程完成local catalog中表的结构优化。
+生产环境中我们通常在yarn集群红使用flink来完成表的结构优化。
```
Comment (YesOrNo828) [inline site/docs/ch/meta-service/dashboard.md:105]: 生产环境中我们通常在yarn集群 **红** 使用flink来完成表的结构优化。
What's the meaning here?

File: site/docs/ch/meta-service/dashboard.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+**1.新增flink类型container**
```
Comment (YesOrNo828) [inline site/docs/ch/meta-service/dashboard.md:107]: which flink version should be chosen?

File: site/docs/ch/meta-service/table-management.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+# 表管理
+[AMS Dashboard](http://localhost:1630)中提供了命令行工具`Terminal`帮组用户快速完成表的创建、修改与删除操作。
+同时还可以在[Spark](#flink.md)和[Flink](#spark/spark-ddl.md)等引擎中使用SQL完成表的管理。
```
Comment (YesOrNo828) [inline site/docs/ch/meta-service/table-management.md:3]: The links between flink and spark are out of order.

File: site/docs/ch/meta-service/table-management.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+## 新建表
+登录[AMS Dashboard](http://localhost:1630)后，进入`Terminal`，输入建表语句并执行即可完成表的创建。
```
Comment (YesOrNo828) [inline site/docs/ch/meta-service/table-management.md:6]: Is there a place to show the default username and password?

**Review by YesOrNo828 — COMMENTED** [review]
@zhoujinsong  I left some comments.

File: site/docs/ch/meta-service/dashboard.md (hunk 1 above)
Comment (zhoujinsong) [reply site/docs/ch/meta-service/dashboard.md:105]: fix it with '生产环境中我们通常在yarn集群**中**使用flink来完成表的结构优化。'

File: site/docs/ch/meta-service/dashboard.md (hunk 2 above)
Comment (zhoujinsong) [reply site/docs/ch/meta-service/dashboard.md:107]: yes, I will add a note to mention this.

File: site/docs/ch/meta-service/table-management.md (hunk 3 above)
Comment (zhoujinsong) [reply site/docs/ch/meta-service/table-management.md:3]: I will fix it

File: site/docs/ch/meta-service/table-management.md (hunk 4 above)
Comment (zhoujinsong) [reply site/docs/ch/meta-service/table-management.md:6]: I mentioned this on the `Arctic Dashboard` page.
User DO need deploy the ams before createing table.

**Review by YesOrNo828 — APPROVED** [review]

**Review by majin1102 — APPROVED** [review]

**Pull request merged**